src/encrypted-metadata/test/helpers/result.js

```diff
@@ -0,0 +1,22 @@
+export * from '@ucanto/core/result'
+import * as API from '@ucanto/interface'
+
+/**
+ * Returns contained `ok` if result is and throws `error` if result is not ok.
+ *
+ * @template T
+ * @param {API.Result<T, {}>} result
+ * @returns {T}
+ */
+export const unwrap = ({ ok, error }) => {
```

Comment:
this isn't built into the js ucanto result type? We should consider adding that to the core library. non-blocking, but something to do.